lua/entities/gmod_wire_cpu.lua

```diff
@@ -140,7 +140,12 @@ local memoryModels = {
 	["flat"]    = {      0, 0      },
 }
 
-function ENT:SetMemoryModel(model)
+function ENT:SetMemoryModel(model,cram,crom)
+	if model == "custom" then
+		self.VM.RAMSize = (tonumber(cram) or 512)*128 -- 65536
+		self.VM.ROMSize = (tonumber(crom) or 512)*128 -- 65536
```

Comment:
Need to clamp the amount with a max

Reply:
Also reminded me that I should be flooring it, thank you.